pkg/util/utils_test.go

```diff
@@ -140,31 +140,86 @@ func TestFriendlyDuration(t *testing.T) {
 	}
 }
 
-func TestCommandExecutesSuccessfully(t *testing.T) {
+func Test_command_executes_successfully_with_absolute_path(t *testing.T) {
 	currentDir, err := os.Getwd()
 	if err != nil {
 		t.Fatalf("Unexpected error: %v", err)
 	}
-	dummybinary := filepath.Join(currentDir, "..", "..", "test", "testdata", "dummybinary")
 
-	if runtime.GOOS == "windows" {
-		dummybinary = filepath.Join(currentDir, "..", "..", "test", "testdata", "dummybinary_windows.exe")
-	}
-	if runtime.GOOS == "darwin" {
-		dummybinary = filepath.Join(currentDir, "..", "..", "test", "testdata", "dummybinary_darwin")
+	dummyBinaryPath := filepath.Join(currentDir, "..", "..", "test", "testdata")
+
+	switch os := runtime.GOOS; os {
+	case "darwin":
```

Comment:
this is treating the OS types are bit more equal and makes the code more regular.